### DISCUSSION TITLE
Able to Use Sidebars For Google Adsense

### DISCUSSION THREAD

**Discussion by cortix** [mmistakes/minimal-mistakes#3534] [Q&A]
Hi all, 

How can i use sidebars for google adsense. for example; I want to use these areas for adsense but I don't want to break down the site as well..  Could you help me?

![Screen Shot 2022-03-23 at 11 47 24](https://user-images.githubusercontent.com/6178404/159659834-2b55f83c-6c0a-4f65-80e5-8b1ac13de931.png)

**Comment by cortix** [comment]


**Reply by iBug** [reply]
Just so you know, it's generally frowned upon to urge others to for specific help in the open-source community. Open-source is run by *people*, motivated by *passion*, and no one gets paid for contributions.

That aside, for the left sidebar, you can look into extending [`_includes/sidebar.html`][1] and adding something near the end. For the right sidebar, it's [`_includes/toc`][2]. What HTML to add is up to you.

  [1]: https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/sidebar.html
  [2]: https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/toc

**Comment by cortix** [comment]
@iBug Thank you for informing. I did not know. I am really sorry
